docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -30,20 +43,23 @@ All BPMN, DMN, and form files contained in the process application folder are de
 If any resource fails to deploy, the whole deployment [fails](#deployment-errors) and the cluster state remains unchanged. This safely ensures that a process application cannot be deployed incompletely or in an inconsistent state.
 :::
 
-## Run a process application
+## Run your process application
+
+You can also manually [run](/components/modeler/web-modeler/run-or-publish-your-process.md#run-a-process) your process application if required, for example to perform manual testing.
 
-You can run your process application to test, debug, and observe how it performs in a live environment.
+:::note
+You should use Play to validate your process application in a development cluster, and only use Run when interacting with other stages such as testing, staging, or production.
```

Comment:
Perhaps simply "Use Play to validate your process application in a development cluster, and only use Run when interacting with other stages such as testing, staging, or production." Or, add the reasoning behind this?